examples/eq_runner_to_downstream/payload_v1/surveyresponse_0_0_1.json

```diff
@@ -29,8 +29,9 @@
     "ref_period_end_date": "2019-10-31"
   },
   "data": {
-    "0001": "This is an answer to q_code 0001",
+    "ABC-123": "This is an answer to q_code ABC-123",
     "0002": "This is an answer to q_code 0002",
-    "0003": "This is an answer to q_code 0003"
+    "xyz1": "This is an answer to q_code xyz1",
+    "ABC-123-xyz": "This is an answer to q_code ABC-123-xyz"
```

Comment:
Don't we want to add an example here that has an underscore if that is something we are now allowing?

Reply:
Added.